templates/post/fileinfo.html

```diff
@@ -22,9 +22,9 @@
 			{% if config.show_filename and file.filename %}
 				, 
 				{% if file.filename|length > config.max_filename_display %}
```

Comment:
Make this more DRY. Move the `{% if file.filename|length > config.max_filename_display %}` inside the `<a>` tag.

Reply:
Updated and tested, amended the previous commit because there's no point merging this tiny thing in two separate commits.

Reply:
sorry friend i don't think you understood what i meant.

the <a> tag can go outside this branch, with the branch only modifying the inner text, which is truncated or not. this way, you don't repeat yourself (DRY). removing a span class is not a good idea as someone's CSS may rely on, so both <span> and <a> can be moved out the branch.